my-site.rb

```diff
@@ -2,7 +2,22 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
```

Comment:
Its still good to have a "root route" so that the user has somewhere to navigate to that doesn't require a path